src/test/java/uk/gov/hmcts/cmc/claimstore/rules/MoreTimeRequestRuleTest.java

```diff
@@ -38,4 +49,38 @@ public void shouldThrowExceptionWhenResponseDeadlinePassed() {
             .build();
         moreTimeRequestRule.assertMoreTimeCanBeRequested(claim);
     }
+
+    @Test
+    public void shouldCallAssertIsNotPastDeadline() {
+        Claim claim = SampleClaim.builder()
+            .withResponseDeadline(LocalDate.now().plusDays(2))
+            .withMoreTimeRequested(false)
+            .build();
+        moreTimeRequestRule.assertMoreTimeCanBeRequested(claim);
+        verify(moreTimeRequestRule).assertIsNotPastDeadline(any(LocalDateTime.class), eq(claim.getResponseDeadline()));
+    }
+
+    @Test
+    public void assertIsNotPastDeadlineShouldNotThrowWhenTimeIsBefore4OnDeadlineDay() {
+        LocalDateTime now = LocalDate.now().atTime(15, 59, 59, 999);
+        LocalDate responseDeadline = LocalDate.now();
+        assertThatCode(
+            () -> moreTimeRequestRule.assertIsNotPastDeadline(now, responseDeadline)
+        ).doesNotThrowAnyException();
+    }
+
+    @Test(expected = MoreTimeRequestedAfterDeadlineException.class)
+    public void assertIsNotPastDeadlineShouldThrowWhenTimeIs4OnDeadlineDay() {
```

Comment:
pm not 0n I assume you meant to have

Reply:
What I wanted to say was _"... when it's 4 on deadline day"_, can add PM here too.